pyproject.toml

```diff
@@ -175,9 +176,11 @@ importlib_metadata = {version = "*", python = "3.8.*"}
 # convenience packages for development
 black = {version = "*", optional = true}
 coverage = {version = "*", optional = true}
+cruft = {version = "*", optional = true}
 mypy = {version = "*", optional = true}
 pre-commit = {version = "*", optional = true}
 pytest = {version = "*", optional = true}
+pytest-mock = {version = "*", optional = true}
```

Comment:
Do we use `pytest-mock` in this repo?

Reply:
No.  Half the repos use it, half don't and I felt weird about overriding so many.